docs/advanced/tls_porting.md

```diff
@@ -24,46 +24,56 @@ mbed TLS distinguishes between strong and weak entropy sources. Of the sources r
 
```

Comment:
I think we need to define and characterise a TRNG. Say what kind of peripheral can use this API and what a TRNG is and some characteristics of it.